packages/remix-dev/index.ts

```diff
@@ -5,6 +5,19 @@ export type { AppConfig, RemixConfig as ResolvedRemixConfig } from "./config";
 export * as cli from "./cli/index";
 
 export type { Manifest as AssetsManifest } from "./manifest";
+export type {
+  DefineRoutesFunction as UNSAFE_DefineRoutesFunction,
+  RouteManifest as UNSAFE_RouteManifest,
+  RouteManifestEntry as UNSAFE_RouteManifestEntry,
+  RouteConfig as UNSAFE_RouteConfig,
+  RouteConfigEntry as UNSAFE_RouteConfigEntry,
+} from "./config/routes";
+export {
+  defineRoutes as UNSAFE_defineRoutes,
+  routeManifestToRouteConfig as UNSAFE_routeManifestToRouteConfig,
+  getRouteConfigAppDirectory as UNSAFE_getRouteConfigAppDirectory,
+} from "./config/routes";
+export { flatRoutes as UNSAFE_flatRoutes } from "./config/flat-routes";
```

Comment:
This isn't needed in React Router because `@remix-run/dev/routes` has access to the internals of the dev package, whereas in this case we need to expose these internals across package boundaries.